en/starter/static-files.md

```diff
@@ -48,7 +48,7 @@ app.use(express.static('files'))
 
 Express looks up the files in the order in which you set the static directories with the `express.static` middleware function.
 
-<div class="doc-box doc-info" markdown="1">NOTE: For best results, [use a reverse proxy](/{{page.lang}}/advanced/best-practice-performance.html#use-a-reverse-proxy) cache to improve performance of serving static assets.
+<div class="doc-box doc-info" markdown="1">For best results, [use a reverse proxy](/{{page.lang}}/advanced/best-practice-performance.html#use-a-reverse-proxy) cache to improve performance of serving static assets.
```

Comment:
Why aren't we using the admonition template here?

Reply:
done